File: agora-easycall-ios-oc/AgoraSample/Menu/AGSAboutViewController.m
//
//  AGSAboutViewController.m
//  AgoraSample
//
//  Created by Shilong on 9/24/15.
//  Copyright (c) 2015 Agora. All rights reserved.
//

#import "AGSAboutViewController.h"
#import "FOWebViewController.h"

@interface AGSAboutViewController ()

@property (weak, nonatomic) IBOutlet UILabel *faqLabel;
@property (weak, nonatomic) IBOutlet UILabel *updateLabel;
@property (weak, nonatomic) IBOutlet UILabel *AgoraLabel;

@end

@implementation AGSAboutViewController

- (void)viewWillAppear:(BOOL)animated
{
    [super viewWillAppear:animated];
    
    //
    self.title = NSLocalizedString(@"About", @"");
    self.AgoraLabel.text = NSLocalizedString(@"Agora Conf.", @"");
    self.updateLabel.text = NSLocalizedString(@"Update", @"");
    self.faqLabel.text = NSLocalizedString(@"FAQ", @"");
}

- (IBAction)didTapFAQView:(id)sender
{
    FOWebViewController *web = [[FOWebViewController alloc] initWithURL:@"http://www.agora.io/faq" title:NSLocalizedString(@"FAQ", @"")];
    [self.navigationController pushViewController:web animated:YES];
}

@end

File: agora-easycall-ios-swift/SampleSwift/Chat/AGSAboutViewController.swift
//
//  AGSAboutViewController.swift
//  SampleSwift
//
//  Created by let on 10/29/15.
//  Copyright © 2015 Agora. All rights reserved.
//

import UIKit

class AGSAboutViewController: UIViewController {
    
    @IBOutlet weak var faqLabel: UILabel!
    @IBOutlet weak var updateLabel: UILabel!
    @IBOutlet weak var AgoraLabel: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }
    
    override func viewWillAppear(animated: Bool) {
        super.viewWillAppear(animated)
        
        //
        title = NSLocalizedString("About", comment: "")
        AgoraLabel.text = NSLocalizedString("Agora Conf.", comment: "")
        updateLabel.text = NSLocalizedString("Update", comment: "")
        faqLabel.text = NSLocalizedString("FAQ", comment: "")
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    

    @IBAction func didTapFAQView(sender: AnyObject) {
        let web = FOWebViewController(URL: "http://www.agora.io/faq", title: NSLocalizedString("FAQ", comment: ""))
        navigationController?.pushViewController(web, animated: true)
    }
    
    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
